pkg/gui/controllers/filtering_menu_action.go

```diff
@@ -90,6 +90,21 @@ func (self *FilteringMenuAction) Call() error {
 		Tooltip: tooltip,
 	})
 
+	menuItems = append(menuItems, &types.MenuItem{
+		Label: self.c.Tr.FilterChangesOption,
+		OnPress: func() error {
+			self.c.Prompt(types.PromptOpts{
+				Title: self.c.Tr.EnterChanges,
+				HandleConfirm: func(response string) error {
+					return self.setFilteringChanges(strings.TrimSpace(response))
```

Comment:
I think we shouldn't trim space now that we support filtering by patch content. Often you actually do care about trailing/leading space when doing that.